scripts/genpotfile

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+usage() { echo "Usage: $0 [-d <po_directory>] <yaml_files>" 1>&2; exit 1; }
+
+while getopts ":d:" opt; do
+    case "${opt}" in
+    d)
+        d=${OPTARG}
+        ;;
+    *)
+        usage
+        ;;
+    esac
+done;
+shift $((OPTIND-1))
+
+if [ -z "${d}" ]; then
+    usage
+fi
+
+toplevel_dir=$(dirname $(readlink -f "$0"))/..
+tmp_dir="/tmp/eos-knowledge-lib/pot"
```

Comment:
I think it would be better to create a directory with `mktemp`, and use a subdir of the build directory, which is after all a temporary directory. Maybe `mktemp -pd`?